homeassistant/components/modbus/services.yaml

```diff
@@ -8,8 +8,8 @@ write_coil:
       required: true
       selector:
         number:
-          min: 1
-          max: 255
+          min: 0
+          max: 65536
```

Comment:
The modbus standard specifies:
In a MODBUS PDU each data is addressed from 0 to 65535.